Mathlib/RingTheory/EuclideanDomain.lean

```diff
@@ -111,3 +111,5 @@ theorem dvd_or_coprime (x y : α) (h : Irreducible x) :
 #align euclidean_domain.dvd_or_coprime EuclideanDomain.dvd_or_coprime
 
 end EuclideanDomain
+
+end
```

Comment:
This entire file is in an anonymous section (not marked `noncomputable`). I'm not sure if that serves any purpose - remove that outer section instead?